client/my-sites/plugins/main.jsx

```diff
@@ -373,57 +378,65 @@ const PluginsMain = React.createClass( {
 					<JetpackManageErrorPage
 						template="optInManage"
 						siteId={ selectedSiteId }
-						title={ this.translate( 'Looking to manage this site\'s plugins?' ) }
+						title={ this.props.translate( 'Looking to manage this site\'s plugins?' ) }
 						section="plugins"
 						featureExample={ this.getMockPluginItems() } />
 				</Main>
 			);
 		}
 
-		const containerClass = classNames( {
-			'main-column': true,
-			plugins: true,
-			'search-open': this.getSearchOpen()
+		const browserUrl = '/plugins/browse' + ( this.props.selectedSiteSlug ? '/' + this.props.selectedSiteSlug : '' );
+		const addPluginButton = <Button
+						href={ browserUrl }
+						className="plugins__header-button"
+						aria-label={ this.props.translate( 'Browse all plugins', { context: 'button label' } ) }>
+						<Gridicon key="plus-icon" icon="plus-small" size={ 18 } />
+						<span className="plugins__header-button-text">{ this.props.translate( 'Add Plugin' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Add a Plugin' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).